client/components/account-balances/balances-tab-panel.tsx

```diff
@@ -71,16 +79,79 @@ const AccountBalancesTabPanel: React.FC = () => {
 			{ ( tab: BalanceTab ) => (
 				<Flex gap={ 0 } className="wcpay-account-balances__balances">
 					<BalanceBlock
-						type="available"
+						id={ `wcpay-account-balances-${ tab.currencyCode }-available` }
+						title={ fundLabelStrings.available }
 						amount={ tab.availableFunds }
 						currencyCode={ tab.currencyCode }
 						isLoading={ isLoading }
+						tooltip={
+							<BalanceTooltip
+								label={ `${ fundLabelStrings.available } tooltip` }
+								content={
+									tab.availableFunds < 0
+										? interpolateComponents( {
+												mixedString: __(
+													'{{learnMoreLink}}Learn more{{/learnMoreLink}} about why your account balance may be negative.',
+													'woocommerce-payments'
+												),
+												components: {
+													learnMoreLink: (
+														<ExternalLink
+															href={
+																documentationUrls.negativeBalance
+															}
+														/>
+													),
+												},
+										  } )
+										: interpolateComponents( {
+												mixedString: __(
+													'The amount of funds available to be deposited. {{learnMoreLink}}Learn More.{{/learnMoreLink}}',
```

Comment:
I haven't been following this review closely so please ignore me if this has been discussed but is this capitalisation correct? Should this be **Learn more** instead of **Learn More**.

Reply:
Ah! Yeh it is meant to be **'Learn more'**. Now I have to do the screenshots again 🤦‍♂️

Thanks for catching this.

Reply:
Updated in c2e52f6a6c61f4ae8c5785f2aa2a681f38af0d9f